test/integration/targets/win_format/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
```

Comment:
Even though we have the alias to skip 2008 and 2008 R2 we still need to check the cmdlet is available and skip the tests itself. This is so people can run the tests locally without it failing. The aliases is just an optimisation for CI so it doesn't spin up a host and waste time/money when it isn't needed. Typically we just check if the cmdlet is available and then include `tests.yml` but you can do this however you like.